hs_core/tasks.py

```diff
@@ -28,19 +29,81 @@
 from hs_core.hydroshare.resource import get_activated_doi, get_resource_doi, \
     get_crossref_url, deposit_res_metadata_with_crossref
 from django_irods.storage import IrodsStorage
-from theme.models import UserQuota, QuotaMessage
+from theme.models import UserQuota, QuotaMessage, UserProfile, User
 from theme.utils import get_quota_message
 
 from django_irods.icommands import SessionException
 
-
 # Pass 'django' into getLogger instead of __name__
 # for celery tasks (as this seems to be the
 # only way to successfully log in code executed
 # by celery, despite our catch-all handler).
 logger = logging.getLogger('django')
 
 
+@periodic_task(ignore_result=True, run_every=crontab(minute=0, hour=0))
+def sync_email_subscriptions():
+    sixty_days = datetime.today() - timedelta(days=60)
+    active_subscribed = UserProfile.objects.filter(email_opt_out=False,
+                                                   user__last_login__gte=sixty_days,
+                                                   user__is_active=True)
+    sync_mailchimp(active_subscribed, "e210a70864")

```

Comment:
We'll probably want these moved to local_settings in case the list changes or we set up a test mailchimp instance for beta or some such thing